client/src/GlobalStyles.js

```diff
@@ -33,8 +32,19 @@ const GlobalStyle = createGlobalStyle`
     h1 {
             font-family: 'Bodoni 72', serif;
             font-weight: 900;
+            font-size: 3.5em !important;
```

Comment:
You should avoid to assign something with !important. Instead, you should inspect your h1 in the browser in order to understand, why the global style css is being overwritten.

Reply:
Thanks Bela! My issue was within the styled components. I mistakenly declared the font size two times. So the styled components overwrote the font size declaration from the GlobalStyes. I deleted the font size from my components and now the `!important` is not needed anymore.